sdk/Trace/Tracer.php

```diff
@@ -23,23 +23,22 @@ class Tracer implements API\Tracer
      */
     private $resource;
 
+    private $rootContext;
+
     public function __construct(
         TracerProvider $provider,
         ResourceInfo $resource,
         API\SpanContext $context = null
     ) {
         $this->provider = $provider;
         $this->resource = $resource;
-        $context = $context ?: SpanContext::generate();
-
-        // todo: hold up, why do we automatically make a root Span?
-        $this->active = $this->generateSpanInstance('tracer', $context);
+        $this->rootContext = $context ? $context : SpanContext::generate(true);
     }
 
     /**
      * @return Span
      */
-    public function getActiveSpan(): API\Span
+    public function getActiveSpan(): ?API\Span
```

Comment:
Docblocks that don't add any information should be removed, I think. In this case there's actually a mismatch between an API Span and an SDK Span o.O

Reply:
Static analysis should have caught that.